web/src/main/java/it/pagopa/selfcare/dashboard/web/controller/InstitutionV2Controller.java

```diff
@@ -76,4 +83,52 @@ public List<InstitutionBaseResource> getInstitutions(Authentication authenticati
 
         return result;
     }
+
+    @PostMapping(value = "/{institutionId}/products/{productId}/users")
+    @ResponseStatus(HttpStatus.CREATED)
+    @ApiOperation(value = "", notes = "${swagger.dashboard.institutions.api.createInstitutionProductUser}")
```

Comment:
you could fill `value` with `${swagger.dashboard.institutions.api.getInstitution}` and add nickname value to set operationId value, for example "v2PostCreateInstitutionProductUser"